potofolio.html

```diff
@@ -201,12 +182,34 @@ <h3>Skills</h3>
                 <li class="list"><button class="but"> Web Development</button></li>
                 <li class="list"><button class="but"> API design</button></li>
             </ul>
-
         </div>
         </section>
 
-    </main>
+        <!-- contact-form -->
 
-    
+    <section id="contact-form">
+        <h1 id="msg" >I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h1>
```

Comment:
- [OPTIONAL] It would be better to use another `hx` tag around the title of this section. The `h1` tag is used for the main title of a webpage and it should not be more than one `h1` tag for better SEO and HTML semantic purposes. Kindly read [this article](https://www.reliablesoft.net/h1-tag/) for more information.